### DISCUSSION TITLE
failed to load translations: failed to unmarshal cs.yaml:

### DISCUSSION THREAD

**Discussion by mysoonyu** [thegeeklab/hugo-geekdoc#561] [Q&A] [answered]
Help need
When i run hugo server with geekdocs template, it complains
hugo server -D
Error: add site dependencies: load resources: loading translations: "/home/vagrant/personalme/hugo/demosite/themes/hugo-geekdoc/i18n/cs.yaml:2:1": failed to load translations: failed to unmarshal cs.yaml: yaml: unmarshal errors:
  line 2: cannot unmarshal !!map into []map[string]interface {}

hugo version
Hugo Static Site Generator v0.68.3/extended linux/amd64 BuildDate: 2020-03-25T06:15:45Z
geekdocs source: https://github.com/thegeeklab/hugo-geekdoc/releases/latest/download/hugo-geekdoc.tar.gz

soonyu

**Comment by xoxys** [comment, marked as answer]
Your Hugo version ` v0.68.3` is pretty outdated and not supported by the theme. The minimum required version is v0.93 while re recommend to always use the latest release.

**Reply by mysoonyu** [reply]
I am using Ubuntu 20.04.4 LTS and install hugo using apt install hugo and it seem only install v0.68.3 version.
Change the installation using snap install hugo and latest version snap:0.108.0 been installed and hugo server is working with geekdocs template
Thanks for suggestion
